src/components/CommitGraph.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { useParams } from '@tanstack/react-router';
+import { getContributors } from '../services/StatsAPI';
+import { IContributor, RouteParams } from '../models';
+import { PolarArea } from 'react-chartjs-2';
+
+const CommitGraph = () => {
+  const {username, repo} : RouteParams = useParams({ strict: false });
+
+  const ContributorQuery = useQuery({
```

Comment:
I see you are only using the data, isLoading and isError results from the useQuery, maybe consider destructuring those out the useQuery. i.e `{data, isLoading, isError} = useQuery(...) `

Reply:
I agree with this, but it depends on your preference.

Reply:
Okay thank you, is it fine if I do it in the next PR?